packages/web/src/components/gcds-file-uploader/readme.md

```diff
@@ -10,6 +10,7 @@
 | `accept`                  | `accept`        | Defines the file types the file uploader accepts.                           | `string`                                            | `undefined` |
 | `disabled`                | `disabled`      | Specifies if a file uploader element is disabled or not.                    | `boolean`                                           | `false`     |
 | `errorMessage`            | `error-message` | Error message for an invalid file uploader element.                         | `string`                                            | `undefined` |
+| `files`                   | --              | FileList of uploaded files to input                                         | `FileList`                                          | `undefined` |
```

Comment:
Any idea why this and the `value` below are not getting generated properly?

Reply:
I just read about it a bit more, and while talking with @ethanWallace found out it's because if it's not a primitive type, [stencil doesn't create an attribute for it unless specified](And I misunderstood - it wasn't about the reflect: true at all! But we do use reflect: true here in file-uploader. Its purpose seems to be so your prop & )